src/bun.js/bindings/bindings.zig

```diff
@@ -1802,6 +1807,106 @@ pub const JSInternalPromise = extern struct {
     };
 };
 
+const AnyPromiseTag = enum { Normal, Internal };
+pub const AnyPromise = union(AnyPromiseTag) {
```

Comment:
`union(enum)` and you can remove `AnyPromiseTag` as a type

Reply:
Done in https://github.com/oven-sh/bun/pull/1697/commits/0323676611ecdefe46d6e334d52dfc7360b2856e